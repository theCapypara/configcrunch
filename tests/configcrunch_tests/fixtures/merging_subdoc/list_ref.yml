base:
    str_field: test
    level_array:
        - $ref: /level1
        - $ref: /level2
        - $ref: /level1
          more:
            merge_test: true
        - name: blub
