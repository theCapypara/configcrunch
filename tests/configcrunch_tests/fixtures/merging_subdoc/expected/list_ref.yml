base:
    str_field: test
    level_array:
        - name: level1
          more:
            merge_test: false
        - name: level2
        - name: level1
          more:
            merge_test: true
        - name: blub